src/test/java/AuthenticationUtilityTest.java

```diff
@@ -125,4 +131,32 @@ public void getAuthHeaderDuplicateTokens() {
     String header = AuthenticationUtility.generateAuthorizationHeader(request);
     Assert.assertNull(header);
   }
+
+  @Test
+  public void getJsonFactory() {
+    // Should return an object that implements the JsonFactory interface
+    Assert.assertTrue(AuthenticationUtility.getJsonFactory() instanceof JsonFactory);
```

Comment:
I don't think this test is necessary as the compiler is already doing this for you. (Same below.)

Reply:
Agreed